app/modules/audit_logs/models.py

```diff
@@ -10,6 +10,8 @@
 
 import uuid
 
+AuditMaxMessageLength = 240
+
 
```

Comment:
I've been thinking a lot about the message length. From the EDM, java stack traces can be enormous, but useful. The exception messages we return from the EDM are pretty small right now (and less useful). We might want to store the longer ones.

Reply:
Agreed. This length limitation is going to be removed as discussed yesterday